tool/planet/start.go

```diff
@@ -559,6 +559,12 @@ func addComponentOptions(config *Config) error {
 
 // addKubeletOptions sets extra kubelet command line arguments in environment
 func addKubeletOptions(config *Config) error {
+	config.Env.Append(EnvKubeletOptions,
+		fmt.Sprintf("--image-credential-provider-config=%v", constants.CredentialProviderConfig))
+	config.Env.Append(EnvKubeletOptions,
+		fmt.Sprintf("--image-credential-provider-bin-dir=%v", constants.CredentialProviderBinDir))
+	config.Env.Append(EnvKubeletOptions, "--feature-gates=KubeletCredentialProviders=true")
+
```

Comment:
Would it make more sense to add these options to the [service start command](https://github.com/gravitational/planet/blob/master/build.assets/makefiles/master/k8s-master/kube-kubelet.service#L12)?

Reply:
That's a good idea as it's hardcoded currently. Fixed in https://github.com/gravitational/planet/pull/883/commits/77212c63e1e91b87fa655de54fa1273ec8164cfc